src/libs/actions/MapboxToken.ts

```diff
@@ -120,13 +113,13 @@ const init = () => {
     }
 
     if (!connectionIDForNetwork) {
-        let network;
+        let network: Network | null;
         connectionIDForNetwork = Onyx.connect({
             key: ONYXKEYS.NETWORK,
             callback: (val) => {
                 // When the network reconnects, check if the token has expired. If it has, then clearing the token will
                 // trigger the fetch of a new one
-                if (network && network.isOffline && val && !val.isOffline && !isCurrentlyFetchingToken && hasTokenExpired()) {
+                if (network?.isOffline && val && !val.isOffline && !isCurrentlyFetchingToken && hasTokenExpired()) {
```

Comment:
Let's rename and use `value` name.
```suggestion
                if (network?.isOffline && val && !value.isOffline && !isCurrentlyFetchingToken && hasTokenExpired()) {
```